internal/admin/dashboard/templates/page-usage.html

```
@@ -0,0 +1,207 @@
+{{define "dashboard-page-usage"}}
+<!-- Usage Page -->
+<template x-if="page==='usage'">
+<div>
+    <div class="page-header">
+        <h2>Usage Analytics</h2>
+        <div class="page-header-controls">
+            <div class="usage-mode-toggle">
+                <button class="usage-mode-btn" :class="{active: usageMode==='tokens'}" @click="toggleUsageMode('tokens')">Tokens</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)